zoo/factories.py

```diff
@@ -42,9 +50,14 @@ class Meta:
         model = Service
 
     owner = Faker("user_name")
-    name = Faker("domain_word")
-    impact = choice(["profit", "customers", "customers"])
-    repository = SubFactory(RepositoryFactory)
+    name = Faker("slug")
+    status = LazyFunction(lambda: choice([item.value for item in Status] + [None]))
+    impact = LazyFunction(lambda: choice([item.value for item in Impact] + [None]))
+    repository = SubFactory(
+        RepositoryFactory, owner=SelfAttribute("..owner"), name=SelfAttribute("..name")
```

Comment:
For now we’ll just create new repositories according to owner name and service name.